website/static/js/filepage/index.js

```diff
@@ -225,41 +221,41 @@ var FileViewPage = {
             m('.btn-group.m-t-xs', [
                 m('.btn.btn-sm.btn-primary.file-download', {onclick: $(document).trigger.bind($(document), 'fileviewpage:download')}, 'Download')
             ]),
-            m('.btn-group.btn-group-sm.m-t-xs', [
-                m('.btn.btn-default.disabled', 'Toggle view: ')
+            m('.btn-group.btn-group-sm.m-t-xs' + (ctrl.editor ? '.btn-group' : ''), [
```

Comment:
Because `.btn-group` is specified in the selector already (the first portion), I'm not sure the ternary here does anything. If it does, please explain. Mithril is still rather opaque to me, even after weeks of looking at it.